lib/PodLogs.py

```diff
@@ -75,7 +76,7 @@ def __init__(self, config_param, wfid):
         log_dir = config_param.args.get("log_dir")
         self._log_dir = os.path.join(log_dir, config_param.timestamp, "argo_logs")
         os.makedirs(self._log_dir, exist_ok=True)
-        self._running_subthreads = []
+        self._running_subthreads = {}
```

Comment:
it's a minor thing, but i think all the "thread" terminology is no longer correct.  i think it would be more correct to say `s/thread/process/`